.github/workflows/scheduled-tasks.yml

```
@@ -0,0 +1,284 @@
+          ./scripts/check-queue.sh
+
+      - name: Alert on failure
+        if: failure()
+        uses: 8398a7/action-slack@v3
+        with:
+          status: failure
+          text: |
+            🚨 Production health check failed!
+            Failed checks: ${{ steps.health.outputs.failed }}
+            Time: ${{ github.event.repository.updated_at }}
+          webhook_url: ${{ secrets.SLACK_WEBHOOK_CRITICAL }}
```

actionlint (1.7.7)

input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "channel", "custom_payload", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "text", "username"

(action)